### PR TITLE
Change login landing page styling

### PR DESCRIPTION
**Pull request by kimchanyoung**
@StevenXL @saraho-shea 
Added preliminary stayling for login page

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -149,3 +149,17 @@ position: absolute;
   box-shadow: none !important;
   text-align: center;
 
+<<<<<<< Updated upstream
```
Comment (StevenXL) [inline app/assets/stylesheets/application.css]: If I'm reading this right, there seems to be remnants of a **merge conflict** here.

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -164,6 +164,17 @@ position: absolute;
...
+}
+
+.info-article {
+  text-align: justify !important;
+}
+
+.info-box {
+  margin-bottom: 2em;
+}
+>>>>>>> Stashed changes
```
Comment (StevenXL) [inline app/assets/stylesheets/application.css]: Not sure what's going on here, but I don't think line 179 or 180 should be included.

**Comment by kimchanyoung** [comment]
Fixed

**Comment by kimchanyoung** [comment]
Merge conflict resolved

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -168,10 +168,12 @@ position: absolute;
   margin-top: 2em;
 }
 
+
 .request-neighborhood {
   margin-top: 2em;
 }
 
+<<<<<<< 43d7512f96fadedb4557ec46ef13cfaa9dda3174
```
Comment (StevenXL) [inline app/assets/stylesheets/application.css]: Should this be here?

**Comment by kimchanyoung** [comment]
sorry -- fixed

**Comment by StevenXL** [comment]
:+1:

**Pull request merged**